lib/theme_check/remote_asset_file.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require "net/http"
+require "pathname"
+require "zlib"
+
+module ThemeCheck
+  class RemoteAssetFile
+    class << self
+      def cache
+        @cache ||= {}
+      end
+
+      def from_src(src)
+        key = uri(src).to_s
+        cache[key] = RemoteAssetFile.new(src) unless cache.key?(key)
+        cache[key]
+      end
+
+      def uri(src)
+        URI.parse(src.sub(%r{^//}, "https://"))
+      end
+    end
+
+    def initialize(src)
+      @uri = RemoteAssetFile.uri(src)
+    end
+
+    def content
+      @content ||= Net::HTTP.get(@uri)
```

Comment:
I think this means we'll fetch every remote access each time we analyze the theme?

And we analyze the theme each time a file is saved in LSP.

This looks like a problem. Maybe we should cache this info in a local file?

Reply:
Aaaah I missed that. Excellent!